OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/DeferredDeepLinkHelper.java

```diff
@@ -28,14 +28,25 @@
 
 import okhttp3.Call;
 import okhttp3.Callback;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
 import okhttp3.Response;
 
 import static android.content.ClipDescription.CLASSIFICATION_COMPLETE;
 import static android.content.Context.CLIPBOARD_SERVICE;
 
 public class DeferredDeepLinkHelper {
+    private static class CachedLink {
```

Comment:
Is regression testing this on the test plan with Ilia?

Reply:
im testing myself 1st, then will send builds to Ilia